src/applications/disability-benefits/all-claims/tests/pages/evidenceTypes.unit.spec.jsx

```diff
@@ -22,75 +27,82 @@ describe('evidenceTypes', () => {
       />,
     );
 
-    expect(form.find('input').length).to.equal(2);
-    form.unmount();
+    expect($$('va-radio-option').length).to.equal(2);
   });
 
   it('should submit when no evidence selected', () => {
     const onSubmit = sinon.spy();
-    const form = mount(
+    const { getByText, container } = render(
       <DefinitionTester
         definitions={formConfig.defaultDefinitions}
         schema={schema}
         uiSchema={uiSchema}
-        data={{
-          'view:hasEvidence': false,
-        }}
+        data={{}}
         formData={{}}
         onSubmit={onSubmit}
       />,
     );
 
-    form.find('form').simulate('submit');
+    $('va-radio', container).__events.vaValueChange({
+      detail: { value: 'N' },
+    });
+    const submitButton = getByText(/submit/i);
+    userEvent.click(submitButton);
+    expect($('va-radio').error).to.be.null;
     expect(onSubmit.calledOnce).to.be.true;
-    expect(form.find('.usa-input-error-message').length).to.equal(0);
-    form.unmount();
   });
 
   it('should require at least one evidence type when evidence selected', () => {
     const onSubmit = sinon.spy();
-    const form = mount(
+    const { getByText, container } = render(
       <DefinitionTester
         definitions={formConfig.defaultDefinitions}
         schema={schema}
         uiSchema={uiSchema}
-        data={{
-          'view:hasEvidence': true,
-        }}
+        data={{}}
         formData={{}}
         onSubmit={onSubmit}
       />,
     );
 
-    form.find('form').simulate('submit');
+    $('va-radio', container).__events.vaValueChange({
+      detail: { value: 'Y' },
+    });
+    const submitButton = getByText(/submit/i);
+    userEvent.click(submitButton);
+    const error = getByText(
+      /please select at least one type of supporting evidence/i,
+    );
+    expect(error).to.exist;
```

Comment:
The `getByText` query throws an exception if there's no match. It should be sufficient to just call `getByText`.